static/view_js/manage.js

```diff
@@ -89,10 +91,102 @@ $(document).ready(function () {
     $("#deleteUserSubmitButton").on("click", function () {
       $("#deleteUser").modal("toggle");
       //Delete user from slot here
+      //var slotKey = createTimeSlotHash(
+      //console.log(userName);
+      var onid = ""
+      var pos = userName.lastIndexOf(" ");
+      var lgn = userName.lngth;
+      onid = userName.slice(pos + 1, lgn);
+      //console.log("pos: " + pos);
+      //console.log("onid: " + onid);
+      //console.log(userTimeSlot);
+      //console.log(hashKey[1]);
+      var newDate = userTimeSlot.replace("on", "");
+      //console.log(newDate);
+      var date = new Date(newDate);
+      //console.log(date);
+      //https://stackoverflow.com/questions/8362952/javascript-output-current-datetime-in-yyyy-mm-dd-hhmsec-format
+      var startTime =
```

Comment:
We might want to look into adding[ Moment.js](https://momentjs.com/) to this project. It'll give us a lot of useful time utilities.